_rules/lists-follow-content-model-a73be2.md

```diff
@@ -0,0 +1,278 @@
+---
+id: a73be2
+name: List elements follow content model
+rule_type: atomic
+description: |
+  This rule checks that list elements have child nodes allowed by the content model
+accessibility_requirements:
+  wcag20:1.3.1: # Info and Relationships (A)
+    forConformance: true
+    failed: not satisfied
+    passed: further testing needed
+    inapplicable: further testing needed
+input_aspects:
+  - Accessibility tree
+  - CSS Styling
+  - DOM Tree
+acknowledgments:
+  authors:
+    - Brian Bors
+    - Wilco Fiers
+  previous_authors:
+    - Jey Nandakumar
+---
+
+## Applicability
+
+This rule applies to any [HTML elements][] with a [hidden state][] of "false", where the [semantic role][] is not changed by an [explicit role][], and for which one of the following is true:
```

Comment:
I understand "semantic role not changed by an explicit role" as `<ul role="list">` being applicable. Is this correct? If so, I think there should be an example with it.
(`<ul role="list">` also applies to "ARIA required owned element", but then maybe these two rules are going to be merged…)